src/microsim/schema/sample/sample.py

```diff
@@ -10,14 +10,18 @@
 from microsim.schema.spectrum import Spectrum
 
 from ._distributions._base import Renderable, RenderableType
-from ._distributions.cosem import CosemLabel
 from ._distributions.direct import FixedArrayTruth
 from ._distributions.matslines import MatsLines
 from .fluorophore import Fluorophore
 
-AnyDistribution = MatsLines | CosemLabel | FixedArrayTruth | RenderableType
-# TODO: this feels like an unDRY hack
-DistributionTypes = MatsLines | CosemLabel | FixedArrayTruth | Renderable
+if TYPE_CHECKING:
+    from ._distributions.cosem import CosemLabel
+
+    AnyDistribution = MatsLines | CosemLabel | FixedArrayTruth | RenderableType
+    DistributionTypes = MatsLines | CosemLabel | FixedArrayTruth | Renderable
+else:
+    AnyDistribution = MatsLines | FixedArrayTruth | RenderableType
+    DistributionTypes = MatsLines | FixedArrayTruth | Renderable
```

Comment:
ah right... hm.  this might make for problems for pydantic, which needs these type hints at runtime.   let me play with this and get back to you